content/recipes/interoperability/bioactivity-profile.md

```diff
@@ -0,0 +1,221 @@
+    1. Project level metadata
+    2. Common sample level metadata such as species, tissue, cell type etc.
+2. Compounds reference with structures
+3. Assay results 
+
+
+### Bioactivity data vocabularies
+
+A set of well-established standards and minimum metadata checklists exist for various aspects of CheMBL formatting:
+
+
+
+* **Chemical information ontology (CHEMINF)** [http://semanticchemistry.github.io/semanticchemistry/ontology/cheminf.owl](http://semanticchemistry.github.io/semanticchemistry/ontology/cheminf.owl) 
+
+    CHEMINF covers information about chemical entities. It includes terms for the descriptors commonly used in cheminformatics software applications and the algorithms which generate them.
+
+* **BioAssay Ontology**(BAO)
+
+    [http://www.bioassayontology.org/bao/bao_complete.owl](http://www.bioassayontology.org/bao/bao_complete.owl) 
+
+
+    The BioAssay Ontology (BAO) describes biological screening assays and their results including high-throughput screening (HTS) data for the purpose of categorizing assays and data analysis. BAO is an extensible, knowledge-based, highly expressive (currently SHOIQ(D)) description of biological assays making use of descriptive logic based features of the Web Ontology Language (OWL). 
+
+* __Ontology of units of Measure (OM)__
+ [http://www.ontology-of-units-of-measure.org/resource/om-2](http://www.ontology-of-units-of-measure.org/resource/om-2) 
+ The OM ontology provides classes, instances, and properties that represent the different concepts used for defining and using measures and units. It includes, for instance, common units such as the SI units meter and kilogram, but also units from other systems of units such as the mile or nautical mile.
+
+
+### Annotation with vocabularies
+
+With the vocabularies above, it’s possible to annotate some metadata fields automatically. For example, the script below annotates “organism: Homo sapiens” with ZOOMA.
+
+
+    ```python
+
+
+    import requests
+    zooma_root = "https://www.ebi.ac.uk/spot/zooma/v2/api/services/annotate?propertyValue="
+    query_keyword = "Homo sapiens"
+    vocab_filter = "&filter=ontology:["
+    vocab_target = "ncbitaxon"
+    query_api = zooma_root + query_keyword + vocab_filter + vocab_target + "]"
+    request_details = requests.get(query_api)
+    request_details.json()
+    ```
+
+
+### Exemplar Bioactivity datasets
+
+[SARS CoV2 phenotypic assay from Caco2 cell line](https://www.ebi.ac.uk/chembl/assay_report_card/CHEMBL4303806/)
+
+The present dataset is a subset of CARE dataset with compounds tested on the Caco2 cell line instead of on the VERO-E6 cell like in the CARE dataset. The dataset can be downloaded and beside structural information it will contain readout numbers for activity (e.g. either percentage of cellular cytopathic inhibition at one concentration or corresponding dose-response extracted IC50 (inhibition concentration at 50% inhibition)).
+
```

Comment:
```suggestion
The present dataset is a subset of [IMI CARE](https://www.imi.europa.eu/projects-results/project-factsheets/care) dataset with compounds tested on the Caco2 cell line instead of on the VERO-E6 cell like in the CARE dataset. The dataset can be downloaded and, besides structural information, it will contain readout numbers for activity (e.g. either `percentage of cellular cytopathic inhibition at a given concentration` or corresponding extracted `dose-response IC50 `(Half-maximal inhibitory concentration)).

```